README.md

```diff
@@ -63,96 +63,96 @@ Supports two mode:
-|                  C#/Mono |    4.421<sub>±0.031</sub> |    20.01<sub>±00.04</sub> + 0.00<sub>±00.00</sub> |     77.47<sub>±03.78</sub> |
-|                  V/clang |    4.676<sub>±0.107</sub> |     0.50<sub>±00.00</sub> + 0.00<sub>±00.00</sub> |    102.59<sub>±03.08</sub> |
-|         Haskell (MArray) |    4.885<sub>±0.166</sub> |     3.72<sub>±00.09</sub> + 1.11<sub>±00.00</sub> |     95.27<sub>±06.35</sub> |
-|               Lua/luajit |    6.784<sub>±0.057</sub> |     2.88<sub>±00.02</sub> + 0.00<sub>±00.00</sub> |    116.90<sub>±02.65</sub> |
-| Ruby/truffleruby (--jvm) |    7.802<sub>±0.288</sub> | 580.68<sub>±15.18</sub> + 679.16<sub>±31.19</sub> |    281.34<sub>±11.68</sub> |
-|         Ruby/truffleruby |    8.594<sub>±0.336</sub> | 414.21<sub>±00.64</sub> + 541.02<sub>±07.15</sub> |    209.20<sub>±18.29</sub> |
-|              Python/pypy |   15.296<sub>±0.235</sub> |   65.41<sub>±00.09</sub> + 45.49<sub>±00.04</sub> |    301.08<sub>±09.78</sub> |
-|                  Haskell |   15.741<sub>±0.410</sub> |     3.79<sub>±00.06</sub> + 0.82<sub>±00.00</sub> |    372.84<sub>±21.06</sub> |
-|                      Lua |   57.706<sub>±2.164</sub> |     2.93<sub>±00.03</sub> + 0.00<sub>±00.00</sub> |   1175.05<sub>±76.72</sub> |
-|             Ruby (--jit) |   61.809<sub>±1.173</sub> |    14.08<sub>±00.02</sub> + 0.23<sub>±00.00</sub> |   1150.99<sub>±65.44</sub> |
-|                     Ruby |   96.539<sub>±2.652</sub> |    13.99<sub>±00.05</sub> + 0.00<sub>±00.00</sub> |   1747.94<sub>±66.65</sub> |
-|               Ruby/jruby |  107.399<sub>±2.175</sub> | 186.72<sub>±01.62</sub> + 130.70<sub>±19.19</sub> |   2264.93<sub>±46.81</sub> |
-|                   Elixir |  118.727<sub>±4.015</sub> |    56.04<sub>±01.20</sub> + 0.00<sub>±00.00</sub> |  2381.04<sub>±129.88</sub> |
-|                   Python |  227.596<sub>±5.242</sub> |    10.34<sub>±00.04</sub> + 0.00<sub>±00.00</sub> |  5115.22<sub>±223.46</sub> |
-|                 Tcl (FP) |  285.989<sub>±6.642</sub> |     4.25<sub>±00.08</sub> + 0.00<sub>±00.00</sub> |  5522.61<sub>±280.47</sub> |
-|                     Perl |  354.862<sub>±6.481</sub> |     6.51<sub>±00.05</sub> + 0.00<sub>±00.00</sub> |  7796.46<sub>±338.55</sub> |
-|                Tcl (OOP) | 544.022<sub>±11.873</sub> |     4.25<sub>±00.08</sub> + 0.00<sub>±00.00</sub> | 11120.82<sub>±814.03</sub> |
+|                 Language |                  Time, s |                                       Memory, MiB |                  Energy, J |
+| :----------------------- | -----------------------: | ------------------------------------------------: | -------------------------: |
+|  Racket (Syntax Objects) |   1.404<sub>±0.026</sub> |   111.62<sub>±00.16</sub> + 0.00<sub>±00.00</sub> |     32.75<sub>±01.55</sub> |
+|                  C++/g++ |   1.446<sub>±0.015</sub> |     1.51<sub>±00.01</sub> + 0.00<sub>±00.00</sub> |     31.37<sub>±00.58</sub> |
```

Comment:
interesting why gcc-11 slower

Reply:
I'm not sure why, but I could confirm that it's definitely an issue with the GCC as I have both 10 and 11, and could see the degradation:

```
root@37056ca3ed45:/src/brainfuck# g++-10 -Wall -O3 -march=native -flto -Wa,-mbranches-within-32B-boundaries -std=c++2a -o target/bin_cpp bf.cpp -I../common/libnotify ../common/libnotify/target/libnotify.a
root@37056ca3ed45:/src/brainfuck# QUIET=1 ../xtime.rb target/bin_cpp bench.b
"C++/g++"
Output checksum: 23280
0.868 s, 1 Mb, 16.2 J

root@37056ca3ed45:/src/brainfuck# g++-11 -Wall -O3 -march=native -flto -Wa,-mbranches-within-32B-boundaries -std=c++2a -o target/bin_cpp bf.cpp -I../common/libnotify ../common/libnotify/target/libnotify.a
root@37056ca3ed45:/src/brainfuck# QUIET=1 ../xtime.rb target/bin_cpp bench.b
"C++/g++"
Output checksum: 23280
1.505 s, 1 Mb, 28.4 J
```
0.868s with g++10 vs 1.505s with g++11